### PR TITLE
feat: try to udpate to Django4

### PR DESCRIPTION
**Pull request by csalom**
Add compatibility with Django >=4.x.

In order to support django versions higher than Django 4, we need to remove support for python <= 3.7

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,9 +25,9 @@ def get_metadata(package, field):
     description='A Django blog app implemented in Wagtail.',
     long_description=codecs.open(os.path.join(os.path.dirname(__file__), 'README.rst'), encoding='utf-8').read(),
     install_requires=[
-        'Django>=3.0,<4.0',
-        'wagtail>=3.0,<5.0',
-        'django-el-pagination>=3.2.4',
+        'Django>=3.8,<4.2',
```
Comment (csalom) [inline setup.py:28]:
```suggestion
        'Django>=4.0,<4.2',
```

File: setup.py [hunk 2]
```diff
@@ -41,13 +41,14 @@ def get_metadata(package, field):
         'Framework :: Django :: 3.0',
         'Framework :: Django :: 3.1',
         'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4.0',
+        'Framework :: Django :: 4.1',
         'Intended Audience :: Developers',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
```
Comment (csalom) [inline setup.py:46]: W can keep this line, we continue supporting python 3.X

File: setup.py (hunk 1 above)
Comment (Brok3r) [reply setup.py]: Done!

File: setup.py (hunk 2 above)
Comment (Brok3r) [reply setup.py:46]: Added!

**Pull request merged**